packages/mobile/src/send/SendConfirmation.tsx

```diff
@@ -66,13 +64,23 @@ const mapDispatchToProps = {
   completePaymentRequest,
 }
 
-const mapStateToProps = (state: RootState): StateProps => {
+const mapStateToProps = (state: RootState, ownProps: NavigationInjectedProps): StateProps => {
+  const { navigation } = ownProps
+  const transactionData = navigation.getParam('transactionData')
```

Comment:
ditto

Reply:
leaving as is for now: https://celo-org.slack.com/archives/CL7BVQPHB/p1589486122236100